### PR TITLE
Loop inside parsed facts

### LINKED ISSUE #10: Use boolean logic to check facts

### PR DESCRIPTION
**Pull request by aozdmr**
Fixes #10

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -12,42 +12,165 @@ log.getLogger('disciplLawReg').setLevel('warn')
...
 describe('discipl-law-reg', () => {
   describe('The discipl-law-reg library', () => {
-    it('correctly parses a single fact', async () => {
-      let parsedFact3 = lawReg.evaluateFactFunction('[fact1]')
+    it('correctly parses and solves a single fact', async () => {
+      let parsedFact = lawReg.evaluateFactFunction('[fact1]')
 
-      expect(parsedFact3).to.deep.equal(
-        'fact1'
+      log.getLogger('disciplLawReg').setLevel('debug')
```
Comment (pimotte) [inline test/index.spec.js]: Let's remove this to not clutter the standard test output (also goes for the other 'setLevel' statements in the other tests)

File: test/index.spec.js [hunk 2]
```diff
@@ -12,42 +12,165 @@ log.getLogger('disciplLawReg').setLevel('warn')
...
+      let ssid = await core.newSsid('ephemeral')
+
+      const factResolver = (fact) => {
+        return fact === '[fact1]'
+      }
+      let result = await lawReg.checkExpression(parsedFact, ssid, { 'factResolver': factResolver })
+
+      expect(result).to.equal(true)
+
+      log.getLogger('disciplLawReg').setLevel('warn')
```
Comment (pimotte) [inline test/index.spec.js]: This line to reset it is than also no longer needed

File: test/index.spec.js [hunk 3]
```diff
@@ -1,53 +1,198 @@
...
 import * as lawReg from '../src/index.js'
-import * as log from 'loglevel'
+// import * as log from 'loglevel'
 
 import awb from './flint-example-awb'
 
 // import { loadConnector } from '../src/connector-loader.js'
 
 // Adjusting log level for debugging can be done here, or in specific tests that need more finegrained logging during development
-log.getLogger('disciplLawReg').setLevel('warn')
```
Comment (pimotte) [inline test/index.spec.js]: I sort of left this here in order to instruct how to use the loglevel, so this is the only one that we would leave. I'm also okay with removing this, but then also remove the comment above it and the import.

Comment (aozdmr) [reply test/index.spec.js]: i added it

**Pull request merged**